style.css

```diff
@@ -3,12 +3,12 @@ body {
   padding: 30px;
   overflow: hidden;
   background-color: #fff;
+  position: absolute;
```

Comment:
Is `position: absolute;` needed?

Reply:
I was testing it but yea I don't think its needed